en/websites_privacy_notice.md

```diff
@@ -16,7 +16,7 @@ We may receive personal information from you based on your interaction with us o
 
 * **Job, Intern, Grant & Fellow Applicants**: Applicants for employment, internship, grant, or fellowship opportunities with Mozilla are required to give us a name, street address, telephone number, email address, and resume, and sometimes additional information as well. We use this information to process and evaluate applications and to communicate with applicants about opportunities. We use [Greenhouse](https://www.greenhouse.io/privacy-policy) to handle employment and intern applications, and [Fluxx.io](https://www.fluxx.io/privacy-policy) to handle fellowship and MOSS grant applications.
 
-* **Contributors**: Volunteering for Mozilla as a community contributor may require Mozilla and others to communicate with you at the email address that you provide in connection to your contribution and to recognize your efforts. If you contribute to Bugzilla, Mozilla Reps, or our code base, then your email address and possibly your name will be publicly available to all internet users. If you create a profile at Mozillians.org, it will be accessible to Mozilla employees and Mozilla contributors. You can edit your profile data at [Profile Settings](https://mozillians.org/user/edit). We sometimes use contributor information from sources (such as Bugzilla) in dashboards to visually share aggregated data on the Mozilla community. An example is [https://wiki.mozilla.org/Contribute/Dashboards](https://wiki.mozilla.org/Contribute/Dashboards). If possible, we try to minimize contact information that is publicly displayed.
+* **Contributors**: Volunteering for Mozilla as a community contributor may require Mozilla and others to communicate with you at the email address that you provide in connection to your contribution and to recognize your efforts. If you contribute to Bugzilla, Mozilla Reps, or our code base, then your email address and possibly your name will be publicly available to all internet users. If you create an account on Mozilla Connect (platform powered by [Khoros](https://khoros.com/privacy), which has its own privacy notice), your name, your avatar, your posts and replies and other information you share will be accessible to Mozilla and other community members. If you create a profile at people.mozilla.org, it will be accessible to Mozilla employees and Mozilla contributors. You can edit your profile data at [Profile Settings](https://mozillians.org/user/edit). We sometimes use contributor information from sources (such as Bugzilla) in dashboards to visually share aggregated data on the Mozilla community. An example is [https://wiki.mozilla.org/Contribute/Dashboards](https://wiki.mozilla.org/Contribute/Dashboards). If possible, we try to minimize contact information that is publicly displayed.
```

Comment:
@mafeld 
This doesn't look right to me.

```
If you create a profile at people.mozilla.org, it will be accessible to Mozilla employees and Mozilla contributors. You can edit your profile data at [Profile Settings](https://mozillians.org/user/edit).
```

That URL redirects, because of the Mozillians->People redirect, but it's 404: https://people.mozilla.org/user/edit
I think the right URL would be https://people.mozilla.org/e?section=personal-info?

Reply:
Thanks. I'll make that change.